packit_service/worker/handlers/bodhi.py

```diff
@@ -159,37 +150,6 @@ def report_in_issue_repository(
             comment_to_existing=body_msg,
         )
 
-    def _error_message_for_auth_error(
-        self, ex: PackitException, koji_build_data: KojiBuildData
-    ) -> str:
-        body = (
-            f"Bodhi update creation failed for `{koji_build_data.nvr}` "
-            f"because of the missing permissions. "
-            f"Please, give {self.service_config.fas_user} user `commit` rights in the "
-            f"[dist-git settings]({self.data.project_url}/adduser). "
-        )
-
-        body += (
-            f"*Try {self.celery_task.retries + 1}/"
-            f"{self.celery_task.get_retry_limit() + 1}."
-        )
-
-        # Notify user on each task run and set a more generous retry interval
-        # to let the user fix this issue in the meantime.
-        if not self.celery_task.is_last_try():
-            body += (
-                f" Task will be retried in "
-                f"{RETRY_INTERVAL_IN_MINUTES_WHEN_USER_ACTION_IS_NEEDED} minutes.*"
-            )
-            self.celery_task.retry(
-                delay=RETRY_INTERVAL_IN_MINUTES_WHEN_USER_ACTION_IS_NEEDED * 60,
```

Comment:
can we remove also the constant or is it used somewhere else?

Reply:
removed; thought it's used in other places where we retry